### DISCUSSION TITLE
Hope to improve the performance of Docker Ollama-CPU, especially for multi-threaded CPU optimization and high memory usage control

### DISCUSSION THREAD

**Discussion by shenyan-008** [open-webui/open-webui#4201] [General]
I am using Windows 10 docker Ollama-CPU. First of all, this is a great project. However, it is not very friendly to multi-threaded CPUs. It can only support 64 CPU threads at most. I have a CPU with 192 threads, but it causes a lot of performance waste when using this project. Can this project optimize multi-threaded CPUs? And the project's memory overhead is too huge. Suppose I actually only need to use 132GB of memory, but the actual memory usage is 200GB, which is equivalent to the maximum memory limit I gave under ".wslconfig"

**Comment by justinh-rahb** [comment]
https://docs.openwebui.com/tutorial/reduce-ram-usage
